apps/voyager-identity-api/src/subgraph.ts

```diff
@@ -26,12 +27,15 @@ export const loadContributions = async (options: {
   limit?: number;
 }): Promise<LDContribution[]> => {
   // Load attestations events from subgraph.
-  const page = options.page || 0; // || for NaN value.
-  const limit = options.limit || LIMIT;
+  const page = Math.abs(options.page || 1); // || for NaN value.
+  const limit = Math.abs(options.limit || LIMIT);
+  const skip = Math.min((page - 1) * limit, SKIP_LIMIT);
```

Comment:
This works for now, but we need to make this pagination more robust. And instead of taking the minimum value we should throw an error. Being explicit is better because then the client can report why their query is failing rather than getting the same values back.